parastell/cubit_io.py

```diff
@@ -61,6 +61,18 @@ def export_step_cubit(filename, export_dir=''):
     export_path = Path(export_dir) / Path(filename).with_suffix('.step')
     cubit.cmd(f'export step "{export_path}" overwrite')
 
+def export_cub5(filename, export_dir=''):
+    """Export cub5 representation of model (native cubit format)
```

Comment:
Use proper grammar in docstrings
```suggestion
    """Export cub5 representation of model (native Cubit format).
```